Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM swift:4.2
+
+WORKDIR /Wall-E
+COPY . /Wall-E
+
+RUN swift build -c release
+
+EXPOSE 2008
```

Comment:
Curious as to why this specific port, is there any reason? (so that I know if I have to do a similar thing in the future)

Did you had to ask DevOps to open a port for the future instance and Wall-E and they just opened this one and told you the port number they'd be opening for Wall-E specifically?

Reply:
@AliSoftware 😛
<img width="663" alt="image" src="https://user-images.githubusercontent.com/1098602/66772698-3c51dd00-eeb5-11e9-8bfb-02d366326a91.png">

But this is not signed-off yet.

Reply:
Ohhh nice easter-egg 😄